server/src/main/scala/org/labrad/browser/RegistryController.scala

```diff
@@ -387,6 +393,11 @@ class RegistryApi @Inject() (cxnHolder: LabradConnectionHolder) {
   }
 
   // callable RPCs
+  
+  @Call("org.labrad.dumb_server.dumb_echo")
+  def dumb_echo(inp: String): Future[String] = {
```

Comment:
use `camelCase` for scala names. The string names for json rpc or labrad settings are fine with underscores.